Standards/scs-XXXX-vN-standard-template.md

```diff
@@ -1,13 +1,15 @@
 ---
 title: _Descriptive title_
-type: Standard  # | Decision Record | Procedural
+type: Standard  # | Procedural
 status: Draft
 track: Global  # | IaaS | Ops | KaaS | IAM
+author: Author <Mail or Github handle>
+date: DD-MM-YYYY
```

Comment:
I just have two questions on this:
1. Why do you want to specify an author? I thought, we want to provide the standards as community so from my point of view, there will not be any single author.
2. Which date should be stated here? The date of the first merge? The date of a standard going from draft to another state?

Thank you for answering these questions

Reply:
I have the same questions.

1. I'm not opposed to stating the authors, but it must be clear how to specify multiple authors.
2. If we do state a date, it must be `yyyy-mm-dd`

Reply:
And *if* we do a date, we'd use YYYY-MM-DD of course.
But indeed, we may not need one ...

Reply:
```suggestion
```